packages/pymedphys_pinnacle/src/pymedphys_pinnacle/export/rtplan.py

```diff
@@ -0,0 +1,548 @@
+                    ds.BeamSequence[beam_count - 1].ControlPointSequence[j].BeamLimitingDeviceRotationDirection = 'NONE'
+                    ds.BeamSequence[beam_count - 1].ControlPointSequence[j].PatientSupportAngle = psupportangle
+                    ds.BeamSequence[beam_count - 1].ControlPointSequence[j].PatientSupportRotationDirection = 'NONE'
+                    ds.BeamSequence[beam_count - 1].ControlPointSequence[j].IsocenterPosition = plan.iso_center
+                    ds.BeamSequence[beam_count - 1].ControlPointSequence[j].GantryRotationDirection = gantryrotdir
+                else:
+                    ds.BeamSequence[beam_count - 1].ControlPointSequence[j].BeamLimitingDevicePositionSequence.append(Dataset()) # This will be the mlcs for control points other than the first
+                    ds.BeamSequence[beam_count - 1].ControlPointSequence[j].BeamLimitingDevicePositionSequence[0].RTBeamLimitingDeviceType = 'MLCX'
+                    ds.BeamSequence[beam_count - 1].ControlPointSequence[j].BeamLimitingDevicePositionSequence[0].LeafJawPositions = leafpositions
+                ds.BeamSequence[beam_count - 1].NumberOfWedges = numwedges # this is temporary value, will read in from file later
+                ds.BeamSequence[beam_count - 1].NumberOfCompensators = '0'# Also temporary
+                ds.BeamSequence[beam_count - 1].NumberOfBoli = '0'
+                ds.BeamSequence[beam_count - 1].NumberOfBlocks = '0'  # Temp
+                ds.BeamSequence[beam_count - 1].BeamLimitingDeviceSequence = Sequence()
+                ds.BeamSequence[beam_count - 1].BeamLimitingDeviceSequence.append(Dataset())
+                ds.BeamSequence[beam_count - 1].BeamLimitingDeviceSequence.append(Dataset())
+                ds.BeamSequence[beam_count - 1].BeamLimitingDeviceSequence.append(Dataset())
+                ds.BeamSequence[beam_count - 1].BeamLimitingDeviceSequence[0].RTBeamLimitingDeviceType = 'ASYMX'
+                ds.BeamSequence[beam_count - 1].BeamLimitingDeviceSequence[1].RTBeamLimitingDeviceType = 'ASYMY'
+                ds.BeamSequence[beam_count - 1].BeamLimitingDeviceSequence[2].RTBeamLimitingDeviceType = 'MLCX'
+                ds.BeamSequence[beam_count - 1].BeamLimitingDeviceSequence[0].NumberOfLeafJawPairs = '1'
+                ds.BeamSequence[beam_count - 1].BeamLimitingDeviceSequence[1].NumberOfLeafJawPairs = '1'
+                ds.BeamSequence[beam_count - 1].BeamLimitingDeviceSequence[2].NumberOfLeafJawPairs = p_count/2
+                bounds = ['-200', '-190', '-180', '-170', '-160', '-150', '-140', '-130', '-120', '-110', '-100', '-95', '-90', '-85', '-80', '-75', '-70', '-65', '-60', '-55', '-50', '-45', '-40', '-35', '-30', '-25', '-20', '-15', '-10', '-5', '0', '5', '10', '15', '20', '25', '30', '35', '40', '45', '50', '55', '60', '65', '70', '75', '80', '85', '90', '95', '100', '110', '120', '130', '140', '150', '160', '170', '180', '190', '200']
```

Comment:
So much of this file contains hard coded linac specific parameters. To work around this for the logfile --> DICOM conversion code I required the user to provide a DICOM template. I think all of the hard coded parameters in this file should be removed, and instead the function should take a template DICOM file which the user provides. Then, the new DICOM file will just be that template DICOM file with the pinnacle results mapped onto it.

See the logic over at https://github.com/pymedphys/pymedphys/blob/master/packages/pymedphys_dicom/src/pymedphys_dicom/delivery/core.py#L157-L186.

I think the first clean up pass through for this code needs to be the removal of any hard coded values.